### ISSUE TITLE
Modifying Generated Tags Pages

### ISSUE THREAD

**Issue by dubrie** [Green-Software-Foundation/green-software-patterns#98]
Hey @osamajandali, I'm wondering if it's possible to manually modify the generated tags pages in any way. Specifically, I like the automated list of patterns within a tag (example: [https://patterns.greensoftware.foundation/tags/compute](https://patterns.greensoftware.foundation/tags/compute)) but would like to include the SME's for each tag on the page as well (example: [https://patterns.greensoftware.foundation/tags/alerting](https://patterns.greensoftware.foundation/tags/alerting)). Since SMEs will be added and removed overtime, is this something that is possible within the docusaurus framework?

**Comment by holanita** [comment]
Hey @dubrie is this sill relevant? Let me know so I can close this or assign Osama accordingly.

**Comment by dubrie** [comment]
No, this can be closed for now.  Going forward it would be nice to have these views more integrated but we can use separate pages for now.
